src/command_modules/azure-cli-interactive/azclishell/__main__.py

```diff
@@ -25,6 +25,10 @@
 
 
 def main(style=None):
+    if APPLICATION.session["interactive"]:
+        print("You're in the interactive shell already!!\n")
+        return
```

Comment:
Don't write this to STDOUT. Use `logger.warning('You're already in the interactive shell.')` which writes to STDERR.

Alternatively, you could just return and do nothing. Less chatty.